### PR TITLE
Chr style

### PR DESCRIPTION
**Pull request by vyepez88**
- Allow fds object to be annotated with either NCBI or UCSC format
- Bug fixes in saving allelic counts

### CODE REVIEW

File: drop/download_data.sh [hunk 1]
```diff
@@ -14,6 +14,9 @@ else
     echo "Data directory not empty, is not updated"
 fi
 
+# move fds object
+mv Data/Output ./
```
Comment (mumichae) [inline drop/download_data.sh]: Why is this necessary?

**Review by mumichae — COMMENTED** [review]
Looks good overall, great to see that we are getting more robust to different chromosome styles. Just a minor question to why we need to move the `Data/Output` directory? Also one of your commits mentioned that you added a new fds object, however I don't seem to see where it is specified and used.

File: drop/download_data.sh (hunk 1 above)
Comment (vyepez88) [reply drop/download_data.sh]: I added a fitted fds with ncbi chr styles to be downloaded with the demo. Then, I copy it to the Output folder resembling the path it would lie on if it had been generated by DROP. Then, I test if gets correctly annotated with gene names even though it has a different chr style than the provided gtf file.

Comment (mumichae) [reply drop/download_data.sh]: I don't quite see where you copy the fds file, though. From this line it looks as if you're copying a whole directory?

Comment (vyepez88) [reply drop/download_data.sh]: yes, I'm copying the whole Output folder, but it only contains the fds file under processed_data/aberrant_splicing ... In that way, it follows the same path and the rule to produce the results recognizes it

File: drop/modules/mae-pipeline/MAE/ASEReadCounter.sh [hunk 2]
```diff
@@ -40,9 +40,10 @@ else
   echo "use UCSC format"
   canonical=$ucsc2ncbi
 fi
-# subset from canonical chromosomes
+
+# subset to standard chromosomes
 chr_subset=$(comm -12 <(cut -f1 -d" " ${canonical} | sort -u) <(echo ${vcf_chr} | xargs -n1 | sort -u))
```
Comment (c-mertes) [inline drop/modules/mae-pipeline/MAE/ASEReadCounter.sh:45]:
```suggestion
chr_subset=$(comm -12 <(cut -f1 -d" " ${canonical} | sort -u) <(echo ${vcf_chr} | xargs -n1 | sort -u))
```
```suggestion
chr_subset=$(comm -12 <(cut -f1 -d" " ${canonical} | sort -u) <(echo ${vcf_chr} | xargs -n1 | sort -u))
```

Comment (c-mertes) [inline drop/modules/mae-pipeline/MAE/ASEReadCounter.sh]:
```suggestion
chr_subset=$(comm -12 <(cut -f1 -d" " ${canonical} | sort -u) <(echo "${vcf_chr}" | sort -u))
```

File: drop/modules/mae-pipeline/MAE/ASEReadCounter.sh [hunk 3]
```diff
@@ -40,9 +40,10 @@ else
...
-chr_subset=$(comm -12 <(echo ${bam_chr} | xargs -n1) <(echo ${chr_subset} | xargs -n1) | uniq)
+chr_subset=$(comm -12 <(echo ${bam_chr} | xargs -n1 | sort -u) <(echo ${chr_subset} | xargs -n1) | uniq)
```
Comment (c-mertes) [inline drop/modules/mae-pipeline/MAE/ASEReadCounter.sh]:
```suggestion
chr_subset=$(comm -12 <(echo "${bam_chr}" | sort -u) <(echo "${chr_subset}") | uniq)
```

File: drop/download_data.sh (hunk 1 above)
Comment (c-mertes) [reply drop/download_data.sh]: This is only needed for testing or? Hence, I would not pollute the user's output folder with such things and have it rather in the testing logic. Basically, when you start testing you can copy it within the pytest code, but not by default.

**Review by c-mertes — CHANGES_REQUESTED** [review]

File: drop/modules/mae-pipeline/MAE/ASEReadCounter.sh (hunk 2 above)
Comment (vyepez88) [reply drop/modules/mae-pipeline/MAE/ASEReadCounter.sh]: changed

File: drop/modules/mae-pipeline/MAE/ASEReadCounter.sh (hunk 3 above)
Comment (vyepez88) [reply drop/modules/mae-pipeline/MAE/ASEReadCounter.sh]: changed

File: drop/download_data.sh (hunk 1 above)
Comment (vyepez88) [reply drop/download_data.sh]: removed

**Review by c-mertes — APPROVED** [review]
looks good now.
